doc/source/index.md

```diff
@@ -0,0 +1,208 @@
+# Welcome to the Ray documentation
+
+```{image} https://github.com/ray-project/ray/raw/master/doc/source/images/ray_header_logo.png
+```
+
+```{image} https://readthedocs.org/projects/ray/badge/?version=master
+:target: http://docs.ray.io/en/master/?badge=master
+```
+
+```{image} https://img.shields.io/badge/Ray-Join%20Slack-blue
+:target: https://forms.gle/9TSdDYUgxYs8SA9e8
+```
+
+```{image} https://img.shields.io/badge/Discuss-Ask%20Questions-blue
+:target: https://discuss.ray.io/
+```
+
+```{image} https://img.shields.io/twitter/follow/raydistributed.svg?style=social&logo=twitter
+:target: https://twitter.com/raydistributed
+```
+
+## What can you do with Ray?
+
+
+````{panels}
+:container: +full-width text-center
+:column: col-lg-4 px-2 py-2
+:card:
+
+**Build distributed applications with <img src="ray-overview/images/ray_svg_logo.svg" alt="ray" width="50px">Core**
+^^^
+Ray Core provides a [simple and flexible API](ray-core/walkthrough.rst) for building and running your distributed applications.
+You can often parallelize single machine code with little to zero code changes.
+
+For instance, with Ray Core you can easily parallelize your functions using 
+[Ray Tasks](ray-core/walkthrough.rst#remote-functions-tasks) or your classes using 
+[Ray Actors](ray-core/actors.rst). 
++++
+```{link-button} ray-core/walkthrough
+:type: ref
+:text: Get Started with Ray Core
+:classes: btn-outline-info btn-block
+```
+---
+**Run machine learning workflows with <img src="ray-overview/images/ray_svg_logo.svg" alt="ray" width="50px">ML**
+^^^
+With Ray ML you can run tasks like [distributed data loading and compute](data/dataset.rst),
+[distributed deep learning](train/train.rst),
+[scalable hyperparameter tuning](tune/index.rst),
+or [industry-grade reinforcement learning](rllib/index.rst).
+
+For production, Ray ML offers [scalable model serving](serve/index.rst) 
+and [fast, durable application flows](workflows/concepts.rst).
+
++++
+```{link-button} ray-overview/index
+:type: ref
+:text: Get Started with Ray ML
+:classes: btn-outline-info btn-block
+```
+
+---
+**Deploy large-scale workloads with <img src="ray-overview/images/ray_svg_logo.svg" alt="ray" width="50px">Cluster**
```

Comment:
```suggestion
**Deploy large-scale workloads with <img src="ray-overview/images/ray_svg_logo.svg" alt="ray" width="50px">Clusters**
```

Reply:
Also consider removing the svg? Not sure if it scales well across devices.

Reply:
It's the size of the font, which does not change. can remove it later if people hate it.